text/0107-construct-library-module-lifecycle.md

```diff
@@ -106,22 +107,21 @@ and related to each other.
       "Working on it" column
     - otherwise, place the module’s tracking issue on the CDK roadmap in the “Working on it“ column
 
+  - **Update Module Stability Label** - In the module's `package.json`, set "Stability": "Experimental" to indicate the
+    high-level constructs are under active development and subject to breaking changes.
```

Comment:
We already set this flag in the package.json as part of the package when it's first created (at stage-0).

This should be merged with the same paragraph mentinoned in the comment against the 'Automatically publish package' item.

Reply:
Good catch.